src/authentication/views/registration-flow/r4-manual-registration.tsx

```diff
@@ -203,7 +202,7 @@ const ManualRegistration: FunctionComponent<ManualRegistrationProps> = ({ histor
 				<Spacer margin="bottom-large">
 					<Button
 						type="secondary"
-						onClick={() => redirectToClientPage(APP_PATH.STAMBOEK.route, history)}
+						onClick={() => history.goBack()}
```

Comment:
Kan je hier niet gewoon `onClick={history.goBack}`? Of heeft goBack optional params?

(Indien dit niet mogelijk is, resolve deze comment maar. :)